py-scripts/create_station.py

```diff
@@ -446,7 +458,12 @@ def build(self):
 
         logger.info("Creating stations")
         self.station_profile.set_command_flag("add_sta", "create_admin_down", 1)
-
+        if self.initial_band_pref.upper() not in ["2.4G", "2G", "5G", "6G"]:
+            logger.info("Initial band preference should be either 2G or 5G or 6G. Aborting...")
+            exit(1)
+        else:
+            band_pref = int(self.initial_band_pref[0])
+            self.station_profile.set_wifi_extra2(initial_band_pref=band_pref)
```

Comment:
Just to double check, if a user doesn't specify a band preference, this logic should set no band preference for the station?